Begrippenlijst.md

```diff
@@ -60,4 +60,7 @@ Staat voor Transmission Control Protocol. Dit protocol wordt gebruikt om een sta
 WebSocket zorgt voor de mogelijkheid om een stroom van berichten in webbrowsers en webservers te sturen bovenop een TCP-verbinding. Lees meer over [WebSocket](https://nl.wikipedia.org/wiki/WebSocket).
 
 #### TMA
-Toegangs Makelaar Amsterdam zorgt ervoor dat wij als amsterdammers kunnen inloggen met digiD. Zorgt voor een veilige connectie tussen 2 clouds, door tussen de 2 clouds te zitten en het in de gaten te houden.
\ No newline at end of file
+Toegangs Makelaar Amsterdam zorgt ervoor dat wij als amsterdammers kunnen inloggen met digiD. Zorgt voor een veilige connectie tussen 2 clouds, door tussen de 2 clouds te zitten en het in de gaten te houden.
+
+#### Response Headers
+De Response Headers laat informatie zien over de webpagina. Zo kun je bijvoorbeeld zien van wat de inhoud is gemaakt en hoelang het is.
```

Comment:
Dit is niet helemaal juist. De headers laten eerder informatie zien over de `server response` en de data die wordt meegestuurd.